samples/accountLinking/skill/states.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const _ = require('lodash');
+
+exports.register = function register(skill) {
+  skill.onIntent('LaunchIntent', (alexaEvent) => {
+    if (_.get(alexaEvent, 'model.user.email')) {
+      return { reply: 'Intent.Launch', to: 'entry' };
+    }
+
+    return { reply: 'Intent.NotAuthenticated', to: 'die' };
+  });
+
+  skill.onIntent('AMAZON.HelpIntent', () => ({ reply: 'Intent.Help' }));
+
+  skill['onAlexaSkillEvent.SkillEnabled']((alexaEvent, reply) => {
```

Comment:
but it does seem like we should be able to handle this event the same way we handle this other custom events

Reply:
I think I got your point, but this is how we are doing it in other skills:

https://github.com/mediarain/mars-skill/blob/staging/skill/events.js#L14
https://github.com/mediarain/vaseline-skill/blob/staging/skill/states/events-state.js#L25